projects/examples/package.json

```diff
@@ -3,14 +3,16 @@
   "main": "src/index.ts",
   "dependencies": {
     "@beanstalk/sdk": "workspace:*",
+    "@beanstalk/sdk-core": "workspace:*",
```

Comment:
Minor thing, but to me "core" implies "foundation" when really these are just common classes, so I'd suggest "sdk-common" as the package name instead.

Reply:
this is a convention used by other web3 projects, like uniswap, so went with it